.devcontainer/README.MD

```diff
@@ -0,0 +1 @@
+The files in this directory configure a development container that can be used for development on Windows using Microsoft's "Dev Containers" extension for VS Code.
```

Comment:
We explicitly removed support for devcontainer because it was hard for us to support and maintain. No one in the maintainers group has experience with it unfortunately.

Let's remove this for now

Reply:
I have been using it since day 1 and will give it support.